styles.css

```diff
@@ -1,66 +1,67 @@
 html {
-    font-size: 10px;
+  font-size: 11px;
 }
 body {
```

Comment:
most HTML elements usually come with their predefined borders and padding. So, I think it should be better if you a global setting of margin:0; and padding:0;. You could also set box-sizing to border box to ensure that every margins are as you define them in the code